### DISCUSSION TITLE
一些图片无法触发画廊模式下的图像预览

### DISCUSSION THREAD

**Discussion by MIKU-N** [AlistGo/alist#372] [Q&A] [answered]
很奇怪，有些jpg可以触发图片预览，有的图片却不可以，明明是同一种图片格式
对于缩略图的图像预览存在大小/格式限制吗？
![image](https://user-images.githubusercontent.com/61493780/149633710-c67fe644-3d31-4859-9f7a-4bf324a85c1d.png)

**Comment by xhofe** [comment, marked as answer]
是什么存储？所有的缩略图都是由对应的存储提供的，只有对应的存储提供了缩略图才会显示缩略图的。

**Reply by MIKU-N** [reply]
储存是SharePoint世纪互联,如何判断其存在缩略图？至少在OneDrive的APP上我是可以看到这些图片的缩略图

**Reply by xhofe** [reply]
你可以F12 看看path请求文件有没有thumbnail 有就是有缩略图，都是直接从OneDrive api拿的

**Reply by MIKU-N** [reply]
我询问了一下世纪互联那边，他们说是微软那边没有给他们上这个功能，所以不是程序问题
![image](https://user-images.githubusercontent.com/61493780/149755531-dad4415b-ec2e-4bc9-836c-8cb89f1989fa.png)

**Reply by xhofe** [reply]
好的 收到
